lib/models/command.js

```diff
@@ -256,60 +256,60 @@ let Command = CoreObject.extend({
     @method validateAndRun
     @return {Promise}
   */
-  validateAndRun(args) {
-    return new Promise((resolve, reject) => {
-      let commandOptions = this.parseArgs(args);
-      // if the help option was passed, resolve with 'callHelp' to call help command
-      if (commandOptions && (commandOptions.options.help || commandOptions.options.h)) {
-        logger.info(`${this.name} called with help option`);
-        return resolve('callHelp');
-      }
+  async validateAndRun(args) {
```

Comment:
Sorry for the whitespace changes.
Had to make this function async to use the `determineInstallCommand` helper.